test/tools/base64_test.cpp

```diff
@@ -16,71 +14,137 @@
  * limitations under the License.
  *
  */
-
-#include "base64.h"
-
 #include <cgreen/cgreen.h>
 
-#include <array>
+#include <memory>
 
-using namespace cgreen;
+#include "binary_serializer.h"
 
-Describe(Base64);
+using io::cloudevents::v1::CloudEvent;
+using namespace cgreen;
 
-BeforeEach(Base64) {
-  // Dummy
+io::cloudevents::v1::CloudEvent cloudEvent;
```

Comment:
CloudEvent  type can be directly used here.

Reply:
Yes, but this change not in this PR.
I will modify it if review team ok to change.

Reply:
@skolli6  - as this is a base64_test and intended to test base64 functionality , why do you need to create cloud events? please just check the base64 functionality

Reply:
Removed cloudevent namespace and it is not applicable for base64 functions.